lib/add_delete.rb

```diff
@@ -0,0 +1,56 @@
+module AddDelete
```

Comment:
Like this use of module to hold additional functionality to ensure single responsibility of classes